tests/ui_tests/test_bundle_list_page.py

```diff
@@ -28,84 +72,228 @@ def _assert_bundle_info_value(attribute: str, actual_info: BundleInfo, expected_
     ), f"Bundle's {attribute} should be {expected_value}, not {actual_value}"
 
 
+# pylint: disable=redefined-outer-name
+@allure.step('Check bundle list is empty')
+def _check_bundle_list_is_empty(page: BundleListPage):
+    assert (
+        row_count := page.table.row_count
+    ) == 0, f'Bundle list should be empty, but {row_count} records was found'
+
+
+@allure.step('Check bundle is listed in table')
+def _open_bundle_list_and_check_info(page: BundleListPage, expected_info: BundleInfo):
+    """
+    Open bundle list page, check that exactly 1 row is presented and check it's info
+    """
+    page.open()
+    assert (
+        row_count := page.table.row_count
+    ) == 1, f'Bundle list should has exactly 1 record, but {row_count} was found'
+    bundle_info = page.get_bundle_info()
+    check_bundle_info_is_equal(bundle_info, expected_info)
+
+
 @allure.step('Check bundle info')
 def check_bundle_info_is_equal(actual_info: BundleInfo, expected_info: BundleInfo):
     for attr in ('name', 'description', 'version', 'edition'):
         _assert_bundle_info_value(attr, actual_info, expected_info)
 
 
-# pylint: disable=redefined-outer-name
 @pytest.fixture()
 def page(app_fs: ADCMTest, login_to_adcm_over_api) -> BundleListPage:
     """Get BundleListPage after authorization"""
     return BundleListPage(app_fs.driver, app_fs.adcm.url).open()
 
 
-@pytest.mark.parametrize(
-    "bundle_archive",
-    [pytest.param(utils.get_data_dir(__file__, "cluster_community"), id="community")],
-    indirect=True,
-)
-def test_ce_bundle_upload(bundle_archive: str, page: BundleListPage):
+@pytest.fixture(params=[[CLUSTER_CE_CONFIG]])
+def create_bundle_archives(request, tmp_path: PosixPath) -> List[str]:
+    """
+    Create dummy bundle archives to test pagination
+    :returns: list with paths to archives
+    """
+    archives = []
+    for i, config in enumerate(request.param):
+        archive_path = tmp_path / f'spam_bundle_{i}.tar'
+        config_fp = (bundle_dir := tmp_path / f'spam_bundle_{i}') / 'config.yaml'
+        bundle_dir.mkdir()
+        with open(config_fp, 'w') as config_file:
+            yaml.safe_dump(config, config_file)
+        with tarfile.open(archive_path, 'w') as archive:
+            archive.add(config_fp, arcname='config.yaml')
+            # assume that ist is declared in first item
+            if 'license' in config[0]:
+                license_fp = os.path.join(utils.get_data_dir(__file__), 'license.txt')
+                archive.add(license_fp, arcname=config[0]['license'])
+        archives.append(str(archive_path))
+    return archives
```

Comment:
Move this fixture to the top-level conftest. Somewhere nearby `create_bundle_archive`

Reply:
Done

Reply:
Done